packages/driver/test/cypress/fixtures/dom.html

```diff
@@ -244,6 +244,7 @@
         <form id="multiple-inputs-and-button-submit">
           <input name="fname" />
           <input name="lname" />
+          <button type="button">button</button>
```

Comment:
here is the updated test btw :+1:

Reply:
probably should have had its own test since this is a regression.

usually i add those and link to the specific issue number just to call out its importance as to what it's fixing.